<launch>
  <!-- start gazebo with an empty plane -->
  <include file="$(find sr_hand)/launch/gazebo/empty_world.launch"/>

  <node pkg="icr" name="model_server" type="model_server" output="screen">
     <param name="icr_prefix" type="string" value="/icr" />  
     <param name="gazebo_prefix" type="string" value="/gazebo" />  
     <param name="model_directory" type="string" value="$(find icr)/models/urdf/"/>
 
  </node>


</launch>
